test/evergreen/evg_cfg.py

```diff
@@ -145,7 +131,7 @@ def get_make_check_dirs():
 
     # Search keyword in CMakeLists.txt to identify directories that involve test configuration.
     # Need to use subprocess 'shell=True' to get the expected shell command output.
-    cmd = "find . -name CMakeLists.txt -exec grep -H -e '\(add_test\|define_c_test|define_test_variants\)' {} \; | cut -d: -f1 | cut -c3- | uniq"
+    cmd = "find . -not -path './releases/*' -name CMakeLists.txt -exec grep -H -e '\(add_test\|define_c_test|define_test_variants\)' {} \; | cut -d: -f1 | cut -c3- | uniq"
```

Comment:
This line of change is to exclude the "releases" folder while figuring out the list of subfolders with test config. The issue was captured while running local testing with contents in the releases folder and is not directly related to the issue reported in this ticket. Include the fix here considering it's a straightforward change.